roles/ceph-common/tasks/installs/install_on_redhat.yml

```diff
@@ -7,7 +7,7 @@
 
 - name: configure ceph yum repository
   include: redhat_ceph_repository.yml
-  when: ceph_origin == 'upstream'
+  when: ceph_origin == 'upstream' or ceph_stable_rh_storage
```

Comment:
Can you please use the following syntax style? (readability purpose)

```
when:
  ceph_origin == 'upstream' or
  ceph_stable_rh_storage
```

Thanks!